pinc/DPage.inc

```diff
@@ -86,15 +90,19 @@ function Pages_prepForRound( $projectid, $round_number )
         // how project_add_page() sets page-states.
 
         $timestamp = time();
-
-        $result = mysqli_query(DPDatabase::get_connection(), "
-            UPDATE $projectid
+        $sql = sprintf("
+            UPDATE %s
             SET
-                state = '{$round->page_avail_state}',
-                {$round->time_column_name} = $timestamp
-        ");
-    
-        _project_adjust_n_available_pages( $projectid, 'ALL' );
+                state = '%s',
+                %s = %d",
```

Comment:
Embed the column name directly, per https://www.pgdp.net/w/index.php?title=DP_Code_Best_Practices#Column_names_as_variables

Reply:
Here and elsewhere.

Reply:
I ended up with some places where I had to build a bunch of temporary strings and concatenate them together to satisfy this, so I replaced them with a a couple of smaller helper functions. Let me know if you dislike the `$setters` code.